content/en/agent/amazon_ecs/_index.md

```diff
@@ -154,14 +154,10 @@ Datadog's [Autodiscovery][14] can be used in conjunction with ECS and Docker to
 
 #### AWSVPC Mode
 
-For Agent v6.10+, `awsvpc` mode is supported for both applicative containers and the Agent container, provided:
+For Agent v6.10+, `awsvpc` mode is supported for applicative containers, provided that security groups are set to allow the host instances security group to reach the applicative containers on relevant ports.
 
-1. For the apps and the Agent in `awsvpc` mode, security groups must be set to allow:
-
-    - The Agent's security group to reach the applicative containers on relevant ports.
-    - The Agent's security group to reach the host instances on TCP port 51678. The ECS Agent container must either run in host network mode (default) or have a port binding on the host.
-
-2. For apps in `awsvpc` mode and the Agent in bridge mode, security groups must be set to allow the host instances security group to reach the applicative containers on relevant ports.
+While it's possible to run the Agent in `awsvpc` mode, it's not the recommneded setup as it may be difficult to retrieve the ENI IP to reach the Agent for Dogstatsd metrics and APM traces.
```

Comment:
```suggestion
While it's possible to run the Agent in `awsvpc` mode, it's not the recommended setup, because it may be difficult to retrieve the ENI IP to reach the Agent for Dogstatsd metrics and APM traces.
```